osf/metadata/utils.py

```diff
@@ -23,6 +23,41 @@
 }
 
 
+def datacite_format_creators(creators):
```

Comment:
Do you think you could combine `datacite_format_creators` and `datacite_format_contributors` to make this more DRY?

Reply:
It's slightly easier to break it up like this because we assuming the contributor is a certain type of contributor call the `ProjectMember` if the feature catches on we will have to break this apart to accommodate additional types and customization, so I want to keep these split, they are different on the schema side.